recipe/meta.yaml

```diff
@@ -150,19 +157,78 @@ outputs:
         - if exist %PREFIX%\\Library\\lib\\parquet_static.lib exit 1        # [win]
         - if exist %PREFIX%\\Library\\lib\\gandiva_static.lib exit 1        # [win]
 
+  - name: arrow-py
+    script: build-pyarrow.sh  # [not win]
+    script: bld-pyarrow.bat   # [win]
+    version: {{ version }}
+    build:
+      number: {{ number }}
+      string: "{{ build_string }}"
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "9.2")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+    requirements:
+      build:
+        - cmake 3.16.*
+        - ninja
+        - make  # [unix]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # pyarrow does not require nvcc but it needs to link against libraries in arrow-cpp=*=*cuda
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+      host:
+        - arrow-cpp ={{ version }}=*{{ build_ext }}
```

Comment:
We should be able now to do `- {{ pin_subpackage('arrow-cpp', exact=True) }}`, can you check whether that works?

Reply:
Yes, it works (tested locally).